crates/bevy_animation/src/util.rs

```diff
@@ -0,0 +1,10 @@
+/// Steps between two different discrete values of any clonable type.
+/// Returns a copy of `b` if `t >= 1.0`, otherwise returns a copy of `b`.
```

Comment:
is it a clone, a copy, or the object itself? comment mentions clone and copy, but signature doesn't bind on those